### PR TITLE
update: Turkey Cities updated

### PR DESCRIPTION
**Pull request by ramazansancar**
Source: https://en.wikipedia.org/wiki/Provinces_of_Turkey

### CODE REVIEW

**Comment by MartinsOnuoha** [comment]
Hi @ramazansancar, thanks for opening an MR to contribute.

File: controllers/countryController.js [hunk 1]
```diff
@@ -108,6 +108,9 @@ class CountryController {
       const { country: countryName } = DB1;
       DB1 = DB1 ? DB1.cities : [];
       DB2 = DB2.states.reduce((acc, state) => acc.concat(state.cities), []).map((x) => x.name);
+      if(country === 'Turkey') {
+        DB2 = [];
+      }
```
Comment (MartinsOnuoha) [inline controllers/countryController.js:113]: Any particular reason why we're doing this?

Comment (ramazansancar) [reply controllers/countryController.js:113]: Sure @MartinsOnuoha, extra data such as districts were included in the cities listed in our country.

Comment (MartinsOnuoha) [reply controllers/countryController.js:113]: Alright, hoping to do an overall refactor for this part soon. For now this is good. I'll merge it 👍

**Review by MartinsOnuoha — APPROVED** [review]

File: controllers/countryController.js (hunk 1 above)
Comment (ramazansancar) [reply controllers/countryController.js:113]: How can I support you with this issue? @MartinsOnuoha

Comment (MartinsOnuoha) [reply controllers/countryController.js:113]: Thanks @ramazansancar  I'll create a different issue for this.

**Pull request merged**